### ISSUE TITLE
What was the cause of the mistake?(start training and then terminate)

### ISSUE THREAD

**Issue by leiq2005** [Unity-Technologies/ml-agents#800]
Hi!
Can you give me some hints, thanks
After the start of training......Excetion........   And then terminate.

INFO:unityagents:Hypermarameters for the PPO Trainer of brain PVEBrain:
        batch_size:     1024
        beta:   0.005
        buffer_size:    10240
        epsilon:        0.2
        gamma:  0.99
        hidden_units:   128
        lambd:  0.95
        learning_rate:  0.0003
        max_steps:      5.0e4
        normalize:      True
        num_epoch:      3
        num_layers:     2
        time_horizon:   64
        sequence_length:        64
        summary_freq:   1000
        use_recurrent:  False
        graph_scope:
        summary_path:   ./summaries/ppo
        memory_size:    256
WARNING:tensorflow:Error encountered when serializing __tensorboard_plugin_asset
__tensorboard_text.
Type is unsupported, or the types of the items don't match field type in Collect
ionDef.
'TextSummaryPluginAsset' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing __tensorboard_plugin_asset
__tensorboard_text.
Type is unsupported, or the types of the items don't match field type in Collect
ionDef.
'TextSummaryPluginAsset' object has no attribute 'name'
INFO:unityagents:Saved Model
WARNING:tensorflow:Error encountered when serializing __tensorboard_plugin_asset
__tensorboard_text.
Type is unsupported, or the types of the items don't match field type in Collect
ionDef.
'TextSummaryPluginAsset' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing __tensorboard_plugin_asset
__tensorboard_text.
Type is unsupported, or the types of the items don't match field type in Collect
ionDef.
'TextSummaryPluginAsset' object has no attribute 'name'
INFO:unityagents:Saved Model
WARNING:tensorflow:Error encountered when serializing __tensorboard_plugin_asset
__tensorboard_text.
Type is unsupported, or the types of the items don't match field type in Collect
ionDef.
'TextSummaryPluginAsset' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing __tensorboard_plugin_asset
__tensorboard_text.
Type is unsupported, or the types of the items don't match field type in Collect
ionDef.
'TextSummaryPluginAsset' object has no attribute 'name'
INFO:unityagents:Saved Model
INFO:unityagents:List of nodes to export :
INFO:unityagents:       action
INFO:unityagents:       value_estimate
INFO:unityagents:       action_probs
INFO:tensorflow:Restoring parameters from ./models/ppo\model-50000.cptk
INFO:tensorflow:Restoring parameters from ./models/ppo\model-50000.cptk
INFO:tensorflow:Froze 10 variables.
INFO:tensorflow:Froze 10 variables.
Converted 10 variables to const ops.
48 ops in the final graph.

**Comment by xiaomaogy** [comment]
Just by the error message it is not enough for us to help you debug. Have you tried our example environment?

**Comment by awjuliani** [comment]
Hi All,

I am closing this thread due to inactivity, but feel free to re-open it if you desire.
